initial (#925)
ealush/vest: website/docs/api_reference.md

- [hasErrors](./writing_your_suite/result_object.md#haserrors-and-haswarnings) - Returns true if the suite or the provided field has errors.
- [hasWarnings](./writing_your_suite/result_object.md#haserrors-and-haswarnings) - Returns true if the suite or the provided field has warnings.

> - [getError](./writing_your_suite/result_object.md#geterror-and-getwarning) - Returns a string error in the suite, or for a specific field.

Reviewer: ```suggestion
- [getError](./writing_your_suite/result_object.md#geterror-and-getwarning) - Returns the first error string of a given field.
```

- [getErrors](./writing_your_suite/result_object.md#geterrors-and-getwarnings) - Returns an object with errors in the suite, or an array of objects for a specific field.

> - [getWarning](./writing_your_suite/result_object.md#geterrors-and-getwarnings) - Returns a string with a warning in the suite, or for a specific field.

Reviewer: ```suggestion
- [getError](./writing_your_suite/result_object.md#geterror-and-getwarning) - Returns the first warning string of a given field.
```
